src/spec/context.cr

```diff
@@ -31,7 +31,8 @@ module Spec
     end
 
     def report(result)
-      Spec.formatter.report(result)
+      Spec.formatters.map{ |formatter| formatter.report(result) }
```

Comment:
No need to [map](http://crystal-lang.org/api/Enumerable.html#map%28%26block%3AT-%3EU%29-instance-method), just `each`. And you can write it shorter like this:

```crystal
Spec.formatters.each &.report(result)
```

The same applies to every line below where you used `map`.